packages/gasket-plugin-intl/generator/intl.ts

```diff
@@ -0,0 +1,20 @@
+/* -- GENERATED FILE - DO NOT EDIT -- */
```

Comment:
This file is no longer needed. Generate support added in https://github.com/godaddy/gasket/pull/831